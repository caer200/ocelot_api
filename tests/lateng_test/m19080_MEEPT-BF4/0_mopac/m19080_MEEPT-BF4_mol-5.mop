CHARGES

 
H	8.712264 +1 5.377359 +1 3.216481 +1
C	8.236679 +1 5.438987 +1 2.396390 +1
C	8.926286 +1 5.256485 +1 1.173148 +1
C	6.891683 +1 5.703888 +1 2.412251 +1
C	8.181416 +1 5.432177 +1 -0.016189 +1
N	10.278841 +1 4.911977 +1 1.161961 +1
H	6.450216 +1 5.823591 +1 3.244884 +1
C	6.158152 +1 5.800588 +1 1.228100 +1
C	6.799852 +1 5.680054 +1 0.023185 +1
S	8.870513 +1 5.357610 +1 -1.590697 +1
C	11.075705 +1 4.883989 +1 0.023775 +1
C	10.900519 +1 4.457014 +1 2.434048 +1
H	5.220230 +1 5.948333 +1 1.257389 +1
H	6.309577 +1 5.764918 +1 -0.786013 +1
C	10.542079 +1 5.105307 +1 -1.271626 +1
C	12.462070 +1 4.606149 +1 0.111116 +1
H	11.519317 +1 3.710456 +1 2.233515 +1
H	10.188643 +1 4.104008 +1 3.024685 +1
C	11.664798 +1 5.529558 +1 3.185583 +1
C	11.360087 +1 5.050148 +1 -2.410786 +1
H	12.863373 +1 4.477335 +1 0.962534 +1
C	13.232784 +1 4.519664 +1 -1.025322 +1
H	12.183357 +1 5.120452 +1 3.922966 +1
H	12.297958 +1 5.983356 +1 2.574541 +1
O	10.741984 +1 6.469719 +1 3.708710 +1
H	10.990216 +1 5.228912 +1 -3.267297 +1
C	12.691088 +1 4.739621 +1 -2.293498 +1
H	14.154964 +1 4.305857 +1 -0.945518 +1
C	11.391651 +1 7.446516 +1 4.533986 +1
H	13.238668 +1 4.674465 +1 -3.067087 +1
H	11.910129 +1 8.073665 +1 3.970099 +1
H	12.016670 +1 6.999262 +1 5.157956 +1
C	10.349221 +1 8.195594 +1 5.304844 +1
H	9.785012 +1 7.558715 +1 5.811249 +1
H	10.784437 +1 8.805841 +1 5.951245 +1
O	9.541325 +1 8.943787 +1 4.404500 +1
C	8.515648 +1 9.655615 +1 5.080731 +1
H	7.983278 +1 9.031477 +1 5.616589 +1
H	7.937750 +1 10.098198 +1 4.424491 +1
H	8.918206 +1 10.329289 +1 5.668022 +1
