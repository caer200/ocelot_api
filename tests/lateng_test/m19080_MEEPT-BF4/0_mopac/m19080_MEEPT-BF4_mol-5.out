 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:31:06 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          8.71226400  *   5.37735900  *   3.21648100  *
     2       C          8.23667900  *   5.43898700  *   2.39639000  *
     3       C          8.92628600  *   5.25648500  *   1.17314800  *
     4       C          6.89168300  *   5.70388800  *   2.41225100  *
     5       C          8.18141600  *   5.43217700  *  -0.01618900  *
     6       N         10.27884100  *   4.91197700  *   1.16196100  *
     7       H          6.45021600  *   5.82359100  *   3.24488400  *
     8       C          6.15815200  *   5.80058800  *   1.22810000  *
     9       C          6.79985200  *   5.68005400  *   0.02318500  *
    10       S          8.87051300  *   5.35761000  *  -1.59069700  *
    11       C         11.07570500  *   4.88398900  *   0.02377500  *
    12       C         10.90051900  *   4.45701400  *   2.43404800  *
    13       H          5.22023000  *   5.94833300  *   1.25738900  *
    14       H          6.30957700  *   5.76491800  *  -0.78601300  *
    15       C         10.54207900  *   5.10530700  *  -1.27162600  *
    16       C         12.46207000  *   4.60614900  *   0.11111600  *
    17       H         11.51931700  *   3.71045600  *   2.23351500  *
    18       H         10.18864300  *   4.10400800  *   3.02468500  *
    19       C         11.66479800  *   5.52955800  *   3.18558300  *
    20       C         11.36008700  *   5.05014800  *  -2.41078600  *
    21       H         12.86337300  *   4.47733500  *   0.96253400  *
    22       C         13.23278400  *   4.51966400  *  -1.02532200  *
    23       H         12.18335700  *   5.12045200  *   3.92296600  *
    24       H         12.29795800  *   5.98335600  *   2.57454100  *
    25       O         10.74198400  *   6.46971900  *   3.70871000  *
    26       H         10.99021600  *   5.22891200  *  -3.26729700  *
    27       C         12.69108800  *   4.73962100  *  -2.29349800  *
    28       H         14.15496400  *   4.30585700  *  -0.94551800  *
    29       C         11.39165100  *   7.44651600  *   4.53398600  *
    30       H         13.23866800  *   4.67446500  *  -3.06708700  *
    31       H         11.91012900  *   8.07366500  *   3.97009900  *
    32       H         12.01667000  *   6.99926200  *   5.15795600  *
    33       C         10.34922100  *   8.19559400  *   5.30484400  *
    34       H          9.78501200  *   7.55871500  *   5.81124900  *
    35       H         10.78443700  *   8.80584100  *   5.95124500  *
    36       O          9.54132500  *   8.94378700  *   4.40450000  *
    37       C          8.51564800  *   9.65561500  *   5.08073100  *
    38       H          7.98327800  *   9.03147700  *   5.61658900  *
    39       H          7.93775000  *  10.09819800  *   4.42449100  *
    40       H          8.91820600  *  10.32928900  *   5.66802200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          8.7123      5.3774      3.2165
     2         C          8.2367      5.4390      2.3964
     3         C          8.9263      5.2565      1.1731
     4         C          6.8917      5.7039      2.4123
     5         C          8.1814      5.4322     -0.0162
     6         N         10.2788      4.9120      1.1620
     7         H          6.4502      5.8236      3.2449
     8         C          6.1582      5.8006      1.2281
     9         C          6.7999      5.6801      0.0232
    10         S          8.8705      5.3576     -1.5907
    11         C         11.0757      4.8840      0.0238
    12         C         10.9005      4.4570      2.4340
    13         H          5.2202      5.9483      1.2574
    14         H          6.3096      5.7649     -0.7860
    15         C         10.5421      5.1053     -1.2716
    16         C         12.4621      4.6061      0.1111
    17         H         11.5193      3.7105      2.2335
    18         H         10.1886      4.1040      3.0247
    19         C         11.6648      5.5296      3.1856
    20         C         11.3601      5.0501     -2.4108
    21         H         12.8634      4.4773      0.9625
    22         C         13.2328      4.5197     -1.0253
    23         H         12.1834      5.1205      3.9230
    24         H         12.2980      5.9834      2.5745
    25         O         10.7420      6.4697      3.7087
    26         H         10.9902      5.2289     -3.2673
    27         C         12.6911      4.7396     -2.2935
    28         H         14.1550      4.3059     -0.9455
    29         C         11.3917      7.4465      4.5340
    30         H         13.2387      4.6745     -3.0671
    31         H         11.9101      8.0737      3.9701
    32         H         12.0167      6.9993      5.1580
    33         C         10.3492      8.1956      5.3048
    34         H          9.7850      7.5587      5.8112
    35         H         10.7844      8.8058      5.9512
    36         O          9.5413      8.9438      4.4045
    37         C          8.5156      9.6556      5.0807
    38         H          7.9833      9.0315      5.6166
    39         H          7.9378     10.0982      4.4245
    40         H          8.9182     10.3293      5.6680

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
